OCPBUGS-21857: release-notes: Customized sshd config need an update (#66331)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

* Some component configuration options and services might have changed between {op-system-base} 8.6 and {op-system-base} 9.2, which means existing machine configuration files might no longer be valid.

> * Notably, if you had customized the default openssh server configuration (`/etc/ssh/sshd_config`), you will have to update it according to link:https://access.redhat.com/solutions/7030537[KB7030537].

Reviewer: ```suggestion
* If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuration file, you must update it according to this link:https://access.redhat.com/solutions/7030537[Red Hat Knowledgebase article].
```
(please do not commit changes with this UI tool, it messes up the ability to squash commits)

* If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.

[discrete]
[id="ocp-4-13-rhcos-ssh-key-location"]
=== Change in SSH key location
{product-title} {product-version} introduces a {op-system-base} 9.2 based {op-system}. Before this update, SSH keys were located in `/home/core/.ssh/authorized_keys` on {op-system}. With this update, on {op-system-base} 9.2 based {op-system}, SSH keys are located in `/home/core/.ssh/authorized_keys.d/ignition`.

> If you had customized the default openssh server configuration (`/etc/ssh/sshd_config`), you will have to update it according to link:https://access.redhat.com/solutions/7030537[KB7030537].

Reviewer: ```suggestion
If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuration file, you must update it according to this link:https://access.redhat.com/solutions/7030537[Red Hat Knowledgebase article].
```
(please do not commit changes with this UI tool, it messes up the ability to squash commits)
